backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/graphicpromotion/internal/BpkGraphicPromoImpl.kt

```diff
@@ -249,6 +302,29 @@ private fun ForegroundContent(
     }
 }
 
+@Composable
+fun getPadding(): Dp =
+    if (isDesktop()) {
+        BpkSpacing.Xxl
+    } else if (isTablet()) {
+        BpkSpacing.Lg
+    } else if (isSmallTablet()) {
+        BpkSpacing.Xl
+    } else {
+        BpkSpacing.Lg
+    }
+
+@Composable
+private fun getDeviceConstrains(): Array<Float> = if (isDesktop()) {
+    arrayOf(RATIO_PORTRAIT_DESKTOP, 436f)
+} else if (isTablet()) {
+    arrayOf(RATIO_PORTRAIT_TABLET, 436f)
+} else if (isSmallTablet()) {
+    arrayOf(RATIO_PORTRAIT_SMALL_TABLET, 617f)
+} else {
+    arrayOf(RATIO_PORTRAIT, 360f)
+}
```

Comment:
Are we able to generate screenshots for tablets and desktop and wether this implementation is tested on a large device?

Reply:
we currently only have one device that's a phone, so right now we don't have an automated way of testing this

Reply:
should I manually post the here in the comments? @marianeum @xo0ps 
I can definitely do that if required